orchestrator/modules/operators/setup.py

```diff
@@ -55,16 +54,17 @@ def load_secrets_from_env(vars_to_load, env_var_dict):
 
 
 def setup_actuators(
-    namespace: str,
-    actuator_configurations: list[ActuatorConfiguration],
+    actuator_configuration_identifiers: list[str],
     discovery_space: DiscoverySpace,
-    queue: MeasurementQueue,
+    measurement_queue: MeasurementQueue,
 ) -> dict[str, "ActuatorActor"]:
     """
+    Creates all the actuators required by discovery_space
+
     Params:
-        namespace: The namespace to set up in
-        config: Configuration of the orchestrator
-        queue: the update queue
+        discovery_space: The discovery space to create the actuators for
+        actuator_configurations: A set of (optional) configurations for actuators in the discoveryspace
```

Comment:
These aren't ActuatorConfigurations anymore, they're identifiers (the variable name has changed as well)